app/src/main/kotlin/com/saltedge/authenticator/features/connections/list/ConnectionsListFragment.kt

```diff
@@ -207,6 +207,17 @@ class ConnectionsListFragment : BaseFragment(),
         viewModel.updateListItemEvent.observe(this, Observer<ConnectionItem> { itemIndex ->
             adapter.updateListItem(itemIndex)
         })
+        viewModel.onShowNoInternetConnectionDialogEvent.observe(this, Observer<ViewModelEvent<GUID>> { event ->
+            event.getContentIfNotHandled()?.let { guid ->
+            activity?.showInfoDialog(
```

Comment:
Why you use different approaches for Delete Connection dialog (DialogFragment) and for No Internet Connection dialog (AlertDialog)?

Reply:
it seemed more convenient in this case, this is a fundamental question?